dependencies.yaml

```diff
@@ -361,7 +383,7 @@ dependencies:
     common:
       - output_types: conda
         packages:
-          - &rmm_conda rmm==24.8.*
+          - &rmm_conda rmm==24.8.*,>=0.0.0a0
```

Comment:
What does this `0.0.0a0` mean?

Reply:
**short answer**

For conda, it's identical to just `==24.8.*`.

For `pip` it means "I'm willing to accept any pre-releases of this package as well".

**long answer**

A version number like that is called a "pre-release".

Those are the version numbers we give to nightly RAPIDS packages.

By default, when you run `pip install cuml-cu12==24.8.*`, such versions will be ignored. That's so package authors can publish packages like release candidates for testing.

One option to tell `pip` you're open to installing such packages is to pass the `--pre` flag to `pip install`, like this:

```shell
pip install --pre 'cuml-cu12==24.8.*`
```

But that can have unintended consequences... `--pre` applies through the entire dependency tree, so running a command like that would also mean you're saying "I'm willing to pull in any pre-releases of any of `cuml-cu12`'s dependencies as well".

Specifying a constraint like `,>=0.0.0a0` like this is a bit stricter... it tells `pip` "I'm willing to pull in any pre-releases of `cuml-cu12` specifically, but not necessarily its dependencies".

There's some more discussion on this in https://peps.python.org/pep-0440/#handling-of-pre-releases

Reply:
We should probably update https://github.com/rapidsai/build-planning/blob/main/docs/docs/packaging.md#nightlies (and then use it as a reference).